### DISCUSSION TITLE
Possible to keep the GoPro from going in sleep-mode after finishing daily script?

### DISCUSSION THREAD

**Discussion by VreekFonq** [gopro/labs#1146] [Q&A]
Is this possible?
Thanks!

**Comment by dnewman-gpsw** [comment]
Sure if you design your script to do that.

**Reply by VreekFonq** [reply]
Any hints. Have searched high and low on the labs pages.
Problem is, having read the documentation, i have trouble connecting the dots on how to script manually (not with standard QR).
Thanks~!

**Reply by dnewman-gpsw** [reply]
You have not offered any information on your application, so it is hard to suggest a starting point.

**Reply by VreekFonq** [reply]
- Video Time-lapse daily from 08:00 to 17:00
- 30sec interval
- every two weeks download of TLV-files on site after stopping time-lapse, preferably with Quik-app and not having to handle the Go Pro 12 itself
- therefore a normal wake state would be nice, in order to download the TLV-files.
- this also keeps the script intact and it will start the next day again without us interfering.

**Reply by dnewman-gpsw** [reply]
There no command to idle, yet !9N will do nothing for 9 seconds. Any longer it will sleep.  Something like 
**>08:00<17:00!S+!17:00E~!9N!R**

**Reply by VreekFonq** [reply]
Hi David, thanks for your answers.
But replying: "Sure if you design your script to do that."
And: "There no command to idle, yet !9N will do nothing for 9 seconds. Any longer it will sleep"
Don't add up to there being a useful function to prevent sleep.
Regards, Ramon

**Reply by dnewman-gpsw** [reply]
The command I shared uses !9N in a loop. It will never sleep. Try it.

**Reply by VreekFonq** [reply]
Ah ok, misunderstood on my part. Will try! Thanks David.

**Reply by VreekFonq** [reply]
PS am i correct thinking the following?
I can put the >08:00<17:00!S+!17:00E~!9N!R command into added QR command blocky and the generate a QR code?

**Reply by dnewman-gpsw** [reply]
Maybe? I'm not sure what you intend to do with it.

**Reply by dnewman-gpsw** [reply]
Note: the whole command can be written in blockly, you don't need to use the "added QR command".
